lametro/models.py

```diff
@@ -417,23 +417,6 @@ def media(self):
         return query_set
 
 
-class LAMetroEventMedia(EventMedia):
```

Comment:
EventMedia queried from `prefetch_related` are not the child class (LAMetroEventMedia). [Apparently, that not yet possible in Django.](85aab34288b5e51fb90031cbaf9616fcfea53ae1) 

So, I moved this to a template filter (for now).